.github/workflows/merge-pytest.yml

```diff
@@ -66,19 +66,19 @@ jobs:
     # Pixel data tests
     - name: Install and test with numpy
       run: |
-        python -m pip install -U numpy
+        python -m pip install --upgrade numpy
         pytest ${{ matrix.pytest-args }}
 
     - name: Install and test Pillow
       run: |
-        python -m pip install -U pillow
+        python -m pip install --upgrade pillow
         pytest ${{ matrix.pytest-args }} pydicom/tests/test_pillow_pixel_data.py
         python -m pip uninstall -y pillow
 
     - name: Install and test jpeg_ls
-      if: ${{ matrix.python-version != '3.11-dev' }}
+      if: ${{ matrix.python-version != '3.12-dev' }}
```

Comment:
Is this in preparation for 3.12 builds?

Reply:
Yes, one more alpha release before the beta cycle starts…

https://peps.python.org/pep-0693/#schedule